.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
I believe we generally don't checkin `.idea/` files into VCS 🤔 @AVI5HEK thoughts?

Reply:
True. Was it added deliberately or auto-created? 🤔 

The root-level gitignore should be enough for the most part. [This](https://github.com/github/gitignore/blob/main/Android.gitignore) template contains the basic setup needed for an Android project.